MANIFEST.in

```diff
@@ -4,3 +4,8 @@ include gimel/VERSION
 include gimel/config.json.template
 include requirements.txt
 recursive-include gimel/vendor *
+
+exclude tox.ini
+exclude requirements.in
+exclude requirements.txt
```

Comment:
it's actually included above... I'm not entirely sure if it's need or not, but we probably want to either exclude or include, but not both :)